src/locales/cy/translation.json

```diff
@@ -196,14 +196,17 @@
             "link_href": "https://exporter.lite.private-beta.service.trade.gov.uk/"
           },
           "RqFZ83csmS4Mi4Y7s7ohD9-ekwU": {
+            "header": "Gwneud cais am wiriad DBS sylfaenol",
```

Comment:
What is the new `header` field used for?

Reply:
The header is used to display content on the delete page.

Reply:
Yes I think I have added it , but not on ofqual .